controller/lights.py

```diff
@@ -0,0 +1,70 @@
+
+from machine import Pin # pylint: disable=E0401
+import neopixel # pylint: disable=E0401
+
+
+class LED_Control:
+
+    def __init__(self, pin, pixelNums):
+        '''
+        Initializes an the control of Led object
+
+        pin (machine class): includes pin for Rasberry Pi Pico W where Led is being used
+        pixelNums (integer): number of Leds that will be used for full strip
+        '''
+
+        self._neopixel = neopixel.NeoPixel(self._pin, self._pixelCount)
+        self._pin = machine.Pin(pin, Pin.OUT)
```

Comment:
Just refer to machine.Pin as Pin, because you used `from machine import Pin` in line 1.